### ISSUE TITLE
Taco verification succeeds but unable to use the connector in Desktop

### ISSUE THREAD

**Issue by roshanrajan-test** [tableau/connector-plugin-sdk#1240]
**About You:**
Name: Roshan
Company: Capgemini

**Your question:**
I have signed a taco file successfully with no error.

The certificate used is a Code Signing Certificate and CA Signed.

When I do a 'jarsigner -verify test.taco', it says that the jar is verified with warning:
**This jar contains entries whose certificate chain is invalid. Reason: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target**

In Tableau Desktop and logs I get the error mentioned in this article and have tried the resolution steps already, no luck:
https://help.salesforce.com/s/articleView?id=001497528&type=1

Please suggest any additional steps I can try to resolve this.

**Comment by pvanderknyff** [comment]
`jarsigner -verify` is not enough, it will verify tacos that are not properly signed. You'll need to run the following command to full validate your taco:
`jarsigner -verify -verbose -certs -strict test.taco`

The -strict flag is the important one. The error you see about the PKIX path building failed will need to be addressed before the taco will load, that is an issue with the certificate and not the Connector SDK.

**Comment by roshanrajan-test** [comment]
Hi @pvanderknyff 
I was able to get it verified with all the switches and command below:
jarsigner -verify -verbose -certs -strict test.taco

Still it doesnt work in Tableau Desktop 2023.1.5.
Checking logs for more clues now.

**Comment by pvanderknyff** [comment]
Yeah, the error you posted shows that something is wrong with the certificate. The strict flag makes sure that it won't be labeled as "verified", without the strict flag jarsigner just checks if the file is signed, not that the signature is valid. The taco will not load until jarsigner with the strict flag shows jar verified and no errors/warnings.

You may need to update your JDK, some warnings are now considered errors in later versions.

**Comment by roshanrajan-test** [comment]
@pvanderknyff 
I was able to fix the error that I was getting when I submitted this issue.
Also as per your suggestion I installed the latest jdk 22 to sign the jar. 
Now the only warning I get is "This jar contains signatures that do not include a timestamp" 
Attached screenshot for refernce.

<img width="574" alt="VerifyWarning" src="https://github.com/tableau/connector-plugin-sdk/assets/69820892/c18f976d-b2ee-4bbb-a48f-01cfcd5861c2">
